### ISSUE TITLE
Accessibility Bug: The Item count information is not conveyed when all the popover is opened.

### ISSUE THREAD

**Issue by puru-hk** [SAP/fundamental-ngx#3141]
Fundamental core component: "Popover"

Description: The Item count information is not conveyed when all the popover is opened.

Expected: The item count information should be conveyed when the ‘Options’ within the popover are accessed. 

Attached screenshot: 

![image](https://user-images.githubusercontent.com/32538291/91651389-ddd02600-eaa9-11ea-8eab-8bf794ca3170.png)

Note: - Issue persists for all the pop over present within the ‘pop over’ control.

**Comment by Sanjay-1312** [comment]
Hello Colleagues,

The count information is read .
On activating the button dropdown will appear the dropdown details are read, It should not be read.
So please fix the issue.

Screenshot:
![image](https://user-images.githubusercontent.com/78714518/120148288-9b3ca500-c205-11eb-885a-0ed14a35c9f9.png)

**Comment by Sanjay-1312** [comment]
Hello colleagues,

Expected Result: On activating the button,  List box,  Available list items (count information). to move to an item press Arrow keys.

**Comment by droshev** [comment]
The popover is just a wrapper and the content inside is all up to the application developer, hence we don't have control.
If you check the menu for example then you will see the proper focus.
